shared/js/background/message-handlers.js

```diff
@@ -140,6 +168,10 @@ export function getLogo () {
     return utils.imgToData('img/social/dax.png')
 }
 
+export function getCloseIcon () {
```

Comment:
Do you mean removing `getCloseIcon()` or replacing the code inside to `getImage()`? I would prefer to not remove `getCloseIcon()` as this avoids that `click-to-load.js` needs to know which image path to provide to `getImage()` to get a close icon. If that's ok, then I'll change this to
```js
export function getCloseIcon () {
    return getImage('img/social/close.svg')
}
```

Reply:
> Do you mean removing getCloseIcon()...

Yea.

> I would prefer to not remove getCloseIcon() as this avoids that click-to-load.js needs to know which image path to provide to getImage() to get a close icon.

Interesting, I'm curious why you want to avoid that?

It's kind of inconsistent at the moment, ~half the image paths being defined in the message handler and the other half being defined in the `click-to-load.js` content script or `social_ctp_configuration.json` configuration. But the way I figure it, the content script sets what the placeholders look like, including their CSS, so why shouldn't it also set which images to use?

All that said, I don't feel too strongly about it. I'm more curious than anything. If you feel strongly, feel free to keep the `getCloseIcon` approach.

Reply:
My reasoning was that no other image path is hardcoded in `click-to-path.js`, they're either defined in `social_ctp_configuration.json` (like specific icons) or are shared images defined in `message-handlers.js` (like `getLoadingImage()`). 

I agree with you that content script sets what the placeholders look like, and that's why images related directly to YT CTL I've added to `social_ctp_configuration.json`.

But since this is a shared close icon (used in the modals by both FB and YT CTL) and that can be used in other places too (it's not specific to the feature, but to the modal), I thought that having it in `message-handlers.js` would fit better with the other use cases, and that's why I prefer to keep it here.

Now, if `message-handlers.js` should have this kind of information, that's another story xD But for the current case, I think it goes better here with the other image message handlers.

Reply:
Fair enough :+1: